hieradata/dev/wso2/wso2is/pattern-1/README.md

```diff
@@ -0,0 +1,17 @@
+# WSO2 IDENTITY SERVER Pattern-1
+
+
+This pattern consist of a stand-alone IS setup with a single node deployment. The databases used in this pattern are the embedded H2 databases. The only difference between this pattern-1 and pattern-2 is that, pattern-1 uses embedded H2 databases and pattern-1 is configured to use external mysql databases.
```

Comment:
A diagram would better explain the deployment, like in https://github.com/wso2/docker-apim/blob/master/docker-compose/pattern-1/README.md